stock_operating_unit/view/stock.xml

```diff
@@ -0,0 +1,196 @@
+        </field>
+    </record>
+
+    <record id="view_location_form" model="ir.ui.view">
+        <field name="name">stock.location.form</field>
+        <field name="model">stock.location</field>
+        <field name="inherit_id" ref="stock.view_location_form"/>
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
+                <field name="operating_unit_id" options="{'no_create': True}"
+                       domain = "[('company_id','=', company_id),
+                       ('user_ids', 'in', uid)]"
+                       groups="operating_unit.group_multi_operating_unit"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_location_tree2" model="ir.ui.view">
+        <field name="name">stock.location.tree</field>
+        <field name="model">stock.location</field>
+        <field name="inherit_id" ref="stock.view_location_tree2"/>
+        <field name="arch" type="xml">
+            <field name="usage" position="after">
+                <field name="operating_unit_id"
+                       domain="[('user_ids', 'in', uid)]"
+                       groups="operating_unit.group_multi_operating_unit"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_location_search" model="ir.ui.view">
+        <field name="name">stock.location.search</field>
+        <field name="model">stock.location</field>
+        <field name="inherit_id" ref="stock.view_location_search"/>
+        <field name="arch" type="xml">
+            <field name="location_id" position="after">
+                <field name="operating_unit_id"
+                       domain="[('user_ids', 'in', uid)]"
+                       groups="operating_unit.group_multi_operating_unit"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="vpicktree" model="ir.ui.view">
+        <field name="name">stock.picking.tree</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.vpicktree"/>
+        <field name="arch" type="xml">
+            <field name="origin" position="after">
+                <field name="operating_unit_id"
+                       domain="[('user_ids', 'in', uid)]"
+                       groups="operating_unit.group_multi_operating_unit"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
+                <field name="operating_unit_id" options="{'no_create': True}"
+                       domain = "[('company_id','=', company_id),
+                       ('user_ids', 'in', uid)]"
+                       groups="operating_unit.group_multi_operating_unit"/>
```

Comment:
IMO, Operating Unit should show in `stock.picking` no `stock.picking.line`

![Selection_122](https://user-images.githubusercontent.com/24691983/78154492-854b1d00-7466-11ea-988e-add79fc611db.png)

This is should be

![Selection_123](https://user-images.githubusercontent.com/24691983/78155079-3651b780-7467-11ea-904b-c0ab44b8ae05.png)

This is my code.

![Selection_124](https://user-images.githubusercontent.com/24691983/78155174-52edef80-7467-11ea-8732-c0906a4c81bf.png)

Reply:
You're totally right. At least in the version 12.0 it as you've suggested. Thanks!